Add command to fix Powershell Problems (#138)
Hochfrequenz/python_template_repository: README.md

has been created.
`.toxbase` is a project independent virtual environment-template for all the tox environments on your machine. If anything is weird during the tox installation or after the installation, try turning your computer off and on again before getting too frustrated.

### Powershell restrictions on Windows
Also on new windows machines it is possible that the execution policy is set to restricted and you are not allowed execute scripts. You can find detailed information [here](https://learn.microsoft.com/de-de/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.3).

> The quickest way to get it working is: Open an Administrator Powershell

Reviewer: ```suggestion
The quickest way to get it working is: Open an Administrator Powershell (e.g. Windows PowerShell App)
```

> The quickest way to get it working is: Open an Administrator Powershell (e.g. Windows PowerShell App)

Reviewer: ```suggestion
The quickest way to solve this problem: Open an Administrator Powershell (e.g. Windows PowerShell App)
```

```ps
Set-ExecutionPolicy -ExecutionPolicy AllSigned
```
and try again (with your regular user, not as admin).

### Creating the project-specifc dev environment.

> If this is the case, clone this repository and create the `dev` environment on your machine.

Reviewer: ```suggestion
If all problems are solved and you're ready to start: 1.  clone the repository, you want to work in 2. create the `dev` environment on your machine. To do this: a) Open an Administrator Powershell (e.g. Windows PowerShell App) b) change directory to your repository c) type in: "tox - e dev"
```

```bash
tox -e dev
```

> The dev environment is the environment which contains both the usual requirements as well as the testing and linting tools.

Reviewer: ```suggestion
You have now created the development environment (dev environment). It is the environment which contains both the usual requirements as well as the testing and linting tools.
```

has been created.
`.toxbase` is a project independent virtual environment-template for all the tox environments on your machine. If anything is weird during the tox installation or after the installation, try turning your computer off and on again before getting too frustrated.

### Powershell restrictions on Windows
Also on new windows machines it is possible that the execution policy is set to restricted and you are not allowed execute scripts. You can find detailed information [here](https://learn.microsoft.com/de-de/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.3).

> If this is the case, clone this repository and create the `dev` environment on your machine.

Reviewer: Wie kommt dsa jetzt hier rein? Ich verstehe diesen Schritt nicht?

Reviewer: Also scheinbar soll das ein Schritt sein, der für die Problemlösung erforderlich ist? Wenn ich das richtig verstanden habe, ist das ok, allerdings verstehe ich dann nicht Zeile 41 - wie hängt die mit 40 zusammen?

Author: das ist rot, wurde entfernt.

Reviewer: achja, ok!
